src/backend.md

```diff
@@ -336,13 +336,25 @@ For serverless projects - method 3 is preferred, but not always. When you have a
      return user;
    }
 
-   // good
+   // also bad
```

Comment:
Let's incorporate this point somehow, to make clear why is it "also bad":

> This does not look like a good example still. I would suggest smth like handleUserCreatedJob here. So we get read of direct side effects in create/update fns and move everything that is not the actual create/update logic out of this functions. The idea is that adding another side effect should never change the original fn

Reply:
done